server/master/src/main/java/org/apache/accumulo/master/tableOps/tableImport/ImportedTableInfo.java

```diff
@@ -38,6 +38,10 @@
   static class DirectoryMapping implements Serializable {
     private static final long serialVersionUID = 1L;
 
+    public DirectoryMapping(String exportDir) {
+      this.exportDir = exportDir;
```

Comment:
Since the `exportDir` is set in the constructor, can we now make the field `final`?